From f30e2d5ee243579afbd14e8f1e33b2318cbad8ce Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Fri, 5 Sep 2014 02:33:19 -0400
Subject: [iommu] Fix compile error in iommu-sysfs.c

Message-id: <20140905023318.497.22189.stgit@gir.stowe>
Patchwork-id: 91382
O-Subject: [RHEL7.1 PATCH 5/7] iommu: Fix compile error in iommu-sysfs.c
Bugzilla: 1136539
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1136539
Upstream Status: ffd78f0093ecd0840ad6436e0522a53defd5df38

commit ffd78f0093ecd0840ad6436e0522a53defd5df38
Author: Joerg Roedel <jroedel@suse.de>
Date:   Mon Jul 7 12:01:21 2014 +0200

    iommu: Fix compile error in iommu-sysfs.c

    Add missing include of <linux/slab.h>.

    Signed-off-by: Joerg Roedel <jroedel@suse.de>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/iommu-sysfs.c b/drivers/iommu/iommu-sysfs.c
index d693923..39b2d91 100644
--- a/drivers/iommu/iommu-sysfs.c
+++ b/drivers/iommu/iommu-sysfs.c
@@ -12,6 +12,7 @@
 #include <linux/device.h>
 #include <linux/iommu.h>
 #include <linux/module.h>
+#include <linux/slab.h>
 
 /*
  * We provide a common class "devices" group which initially has no attributes.
-- 
1.7.1